src/app/pages/homepage/homepage.html

```diff
@@ -1,5 +1,5 @@
 <header class="docs-header-background"
-        [class.is-next-version]="isNextVersion">
+        [class.is-next-version]="isNextVersion" id="main-content" role="main" aria-label="Get started">
```

Comment:
On other pages the `<main>` element is the `#main-content`, is this a mistake?

Reply:
I wasn't sure whether to put it here or in main because there are 2 "Get Started" buttons and this one was the first one

Reply:
It doesn't seem expected or ideal for a `header` element to have `role="main"`.

Reply:
Ok this makes sense after I looked at the page. Like Michael said, I would just remove the `role="main"`